code/lib/csf-tools/src/enrichCsf.ts

```diff
@@ -2,21 +2,81 @@
 import * as t from '@babel/types';
 // eslint-disable-next-line import/no-extraneous-dependencies
 import * as generate from '@babel/generator';
-// eslint-disable-next-line import/no-extraneous-dependencies
-import * as template from '@babel/template';
 import type { CsfFile } from './CsfFile';
 
-export const enrichCsf = (csf: CsfFile) => {
+export interface EnrichCsfOptions {
+  disableSource?: boolean;
+  disableDescription?: boolean;
+}
+
+export const enrichCsf = (csf: CsfFile, options?: EnrichCsfOptions) => {
   Object.keys(csf._storyExports).forEach((key) => {
     const storyExport = csf.getStoryExport(key);
-    const source = extractSource(storyExport);
-    const addParameter = template.default(`
-       %%key%%.parameters = { storySource: { source: %%source%% }, ...%%key%%.parameters };
-    `)({
-      key: t.identifier(key),
-      source: t.stringLiteral(source),
-    }) as t.Statement;
-    csf._ast.program.body.push(addParameter);
+    const source = !options?.disableSource && extractSource(storyExport);
+    const description =
+      !options?.disableDescription && extractDescription(csf._storyStatements[key]);
+    const parameters = [];
+    // storySource: { source: %%source%% },
```

Comment:
Did this mean to stay here? -- or maybe intended down a few lines, above `if (source)`?